lecture/nn/nn2_linear_regression.md

```diff
@@ -35,6 +35,19 @@ youtube:
 attachments:
   - link: "https://raw.githubusercontent.com/Artificial-Intelligence-HSBI-TDU/KI-Vorlesung/master/lecture/nn/files/NN2-Lineare_Regression.pdf"
     name: "NN2-Lineare_Regression.pdf"
+challenges: |
+    **Skalierung der Merkmale**
+
+    Abbildung 1 und Abbildung 2 zeigen die [Höhenlinien](https://de.wikipedia.org/wiki/H%C3%B6henlinie) ([Contour Lines](https://en.wikipedia.org/wiki/Contour_line)) von zwei Kostenfunktionen.
+ 
+    ![Abbildung 1](images/contour_plot_a.png){width="40%"}
+    ![Abbildung 2](images/contour_plot_b.png){width="40%"}
```

Comment:
@cyildiz Diese beiden Abbildungen scheinen noch in "lecture/nn/images/" zu fehlen?